src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,52 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_SIZE = 10;
+    private K[] keys = (K[]) new Object[MAX_SIZE];
+    private V[] values = (V[]) new Object[MAX_SIZE];
+    private int size = 0;
```

Comment:
```suggestion
    private int size;
```

`size` variable will be initialized by 0 by default